roles/tpa_single_node/vars/main.yml

```diff
@@ -20,9 +20,16 @@ tpa_single_node_pgsslmode: disable
 
 # S3_SERVICE
 tpa_single_node_s3_url:
-tpa_single_node_s3_username:
```

Comment:
It's preferable not to store sensitive values in configuration files.
For now we might want to use envars.
This applies to any username, password or secret below.

Reply:
Agree! updated with environment variable lookup.